api/build.gradle

```diff
@@ -20,7 +20,7 @@ dependencies {
         exclude group: 'com.google.errorprone', module: 'error_prone_annotations'
         // prefer 3.0.2 from libraries instead of 3.0.1
         exclude group: 'com.google.code.findbugs', module: 'jsr305'
-        // prefer 1.17 from libraries instead of 1.14
+        // prefer our own version from libraries instead of Guava's dependency
```

Comment:
[Guava 28.1](https://search.maven.org/artifact/com.google.guava/guava-parent/28.1-jre/pom) has dependency animal-sniffer-annotation 1.18. The old comment was outdated; it was hard to maintain the comment precisely reading the Guava's dependency etc.

@creamsoup  I updated this comment and now feel these comments "prefer XXX from libraries instead of YYY" is hard to maintain and unnecessary. Don't you think we should remove them?

Reply:
yeah it is hard to maintain, i like your new comment (or no comment is also good enough)